parrots.json

```diff
@@ -576,5 +576,9 @@
       "hd": "hd/redenvelopeparrot.gif",
       "name": "Red Envelope Parrot",
       "tip": "Happy Lunar New Year!"
+    },
+    {
+      "hd": "hd/partysloth.gif",
```

Comment:
Looks like there is a name mismatch from `sloth.gif` to `partysloth.gif`.

Reply:
Oh, d'oh! Good catch. Too much partysloth excitement to think clearly on this Friday evening.